std/typecons.d

```diff
@@ -1615,6 +1615,24 @@ if (distinctFieldNames!(Specs))
     assert(!(t == t));
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=8015
+@safe unittest
+{
+    import std.typecons : Tuple;
```

Comment:
This import is no longer required, right?

Reply:
You are right...